src/pages/Login.tsx

```diff
@@ -134,17 +156,29 @@ const Login = (): JSX.Element => {
 
                   <div className="-mt-px">
                     <Field
+                      data-testid="password"
                       aria-label="Password"
                       name="password"
                       type="password"
-                      required
                       className="appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-b-md focus:outline-none focus:shadow-outline-blue focus:border-blue-300 focus:z-10 sm:text-sm sm:leading-5"
                       placeholder="Password"
                     />
                   </div>
 
                   <div className="mt-6">
-                    <LoginButton loading={formik.isSubmitting}>Sign In</LoginButton>
+                    <LoginButton loading={formik.isSubmitting} data-testid="signIn" disabled={formik.isSubmitting}>
+                      Sign In
+                    </LoginButton>
+                  </div>
```

Comment:
Just a UI thought, Would it be better to add an 'Or' in between Register and Sign In and to mention in  smaller font/Italics for 'Existing users' near to Sign In ?